doc/examples/transform/plot_ransac.py

```diff
@@ -114,3 +117,53 @@
            marker='o', label='Outlier data')
 ax.legend(loc='lower left')
 plt.show()
+
+######################################################################
+# Circle model
+# ============
+from skimage.measure import CircleModel
+
+
+def generate_noisy_circle(size, noise_level, radius=1):
+    angle = 2 * np.pi * np.random.rand(size)
+    rad = radius + noise_level * (2 * np.random.rand(size) - 1)
+    x = rad * np.cos(angle)
+    y = rad * np.sin(angle)
+    return x, y
+
+
+# Generate three types of data points
+x1, y1 = generate_noisy_circle(1000, 0.001)
+x2, y2 = generate_noisy_circle(1000, 0.1)
+half_width = 1.5
+x3, y3 = (2 * half_width * np.random.rand(1000) - half_width,
+		  2 * half_width * np.random.rand(1000) - half_width)
+
+x, y = np.concatenate((x1, x2, x3)), np.concatenate((y1, y2, y3))
+xy = np.stack([x, y], axis=1).astype(float)
+
+# Fit with RANSAC
+ransac_kwarg = {'min_samples': 100,
+                'residual_threshold': .01,
+                'max_trials': 1000, }
+model_robust, inliers = ransac(xy, CircleModel, **ransac_kwarg)
+outliers = inliers == False
+cy, cx, r = model_robust.params
+
+
+fig, ax = plt.subplots(ncols=2, figsize=(16, 7))
+
+ax[0].set_title('Input data')
+ax[0].plot(x3, y3, '.')
+ax[0].plot(x2, y2, '.')
+ax[0].plot(x1, y1, '.')
+
+circle = plt.Circle((cy, cx), radius=r, alpha=0.1)
+ax[1].add_patch(circle)
+ax[1].plot(xy[inliers, 0], xy[inliers, 1],
+           'bo', markersize=1, label='inliers')
+ax[1].plot(xy[~inliers, 0], xy[~inliers, 1],
```

Comment:
`outliers` was defined above. Did you intend to use that here instead of `~inliers`?